includes/admin/class-rop-global-settings.php

```diff
@@ -621,4 +626,24 @@ public function get_all_services_handle() {
 
 		return $all;
 	}
+
+	/**
+	 * Check license is new or not.
+	 *
+	 * @return bool
+	 */
+	public function check_is_new_license() {
+		// Check is valid license.
+		if ( $this->license_type() > 0 ) {
+			$product_key  = 'tweet_old_post_pro';
+			$license_data = get_option( $product_key . '_license_data', '' );
+			if ( isset( $license_data->created_at ) ) {
+				$created_at = strtotime( date( 'Y-m-d', strtotime( $license_data->created_at ) ) );
+				$compare_to = strtotime( '2024-07-21' );
+				return $created_at > $compare_to;
+			}
+		}
+
+		return false;
```

Comment:
NIT. Just a personal style, we can make this look nicer:

```php
public function check_is_new_license() {
    if ( $this->license_type() <= 0 ) { // Ignore free users.
        return false;
    }

    $product_key = 'tweet_old_post_pro';
    $license_data = get_option($product_key . '_license_data', '');

    if ( ! isset( $license_data->created_at ) ) {
        return false;
    }

    // Is after 9.1?
    $created_at = strtotime(date('Y-m-d', strtotime($license_data->created_at)));
    $compare_to = strtotime('2024-07-21');

    return $created_at > $compare_to;
}
```

Reply:
@Soare-Robert-Daniel Thank you for the review, I've applied the provided code with my latest commit.